manifests/config/exporter/patch/patch-openshift.yaml

```diff
@@ -19,15 +19,10 @@ spec:
         volumeMounts:
         - name: kernel-src
           mountPath: /usr/src/kernels
-        - name: kernel-debug
-          mountPath: /sys/kernel/debug
+          readOnly: true
```

Comment:
@sunya-ch can we remove the kernel debug path with libbpf?
Did you test that?

Reply:
https://github.com/libbpf/libbpf/blob/master/src/libbpf.c#L10319 seems to indicate that it will be useful to have this mounted.

Reply:
if above is true, maybe we need think about add some comment so we won't remove this later

Reply:
I tried it before when working on https://github.com/sustainable-computing-io/kepler-operator/pull/185 with both latest and latest-libbpf and I tested by running the operator locally and installing kepler + the Grafana dashboard without the /sys/kernel/debug mount. I could still see metrics in Grafana with both latest and latest-libbpf images if I recall correctly so it should work from my understanding.
If you know how we can test this more and better please tell me!
Also when I have time I will test this again just to be sure!

Reply:
> @sunya-ch can we remove the kernel debug path with libbpf?
> Did you test that?

sorry for delays. It is fine to remove it here because it is mounted at the beginning in the exporter ds. No need to patch.

Reply:
@sunya-ch where is it mounted?
Are you saying that this file is just duplicating something in another file?

Reply:
`/sys` path is mounted here if we deploy exporter via make manifest: https://github.com/sustainable-computing-io/kepler/blob/5272ee2f8f6ab0afc4214884ba143b6733e53f61/manifests/config/exporter/exporter.yaml#L112-L114.

patch-openshift.yaml is supposed to include only the patch that is required specific to openshift not general.

Reply:
@marceloamaral thanks for the ping. Noted, I will compare the daemonset again to ensure that they are in sync.